solidity/contracts/bridge/Bridge.sol

```diff
@@ -32,12 +51,23 @@ pragma solidity 0.8.4;
 ///         about the sweep increasing appropriate balances in the Bank.
 /// @dev Bridge is an upgradeable component of the Bank.
 contract Bridge {
+    using BTCUtils for uint256;
+    using BTCUtils for bytes;
+    using BytesLib for bytes;
+    using ValidateSPV for bytes;
+    using ValidateSPV for bytes32;
+
     struct DepositInfo {
         uint64 amount;
         address vault;
         uint32 revealedAt;
     }
 
+    // confirmations on the Bitcoin chain
```

Comment:
Let's add a `@notice` comment to `TX_PROOF_DIFFICULTY_FACTOR` and add a blank line after it.

Reply:
Done.